frontend/admin/src/js/components/classification/ClassificationTable.tsx

```diff
@@ -112,7 +113,16 @@ export const ClassificationTable: React.FC<
 };
 
 export const ClassificationTableApi: React.FunctionComponent = () => {
-  const [result] = useGetClassificationsQuery();
+  const context = useMemo<Partial<OperationContext>>(
+    () => ({
+      additionalTypenames: ["Classification"], // This lets urql know when to invalidate cache if request returns empty list. https://formidable.com/open-source/urql/docs/basics/document-caching/#document-cache-gotchas
+      requestPolicy: "cache-first", // The list of classifications will rarely change, so we override default request policy to avoid unnecessary cache updates.
+    }),
+    [],
+  );
```

Comment:
It is best to only use `useMemo` for complex computations since this can result in a lack of garbage collection. This could also result in a performance hit since it needs to execute a function and assign values taking longer than just assigning the values.

REF: https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
You're right. I copied the use of useMemo from the [urql docs](https://formidable.com/open-source/urql/docs/basics/document-caching/#adding-typenames), but I can't imagine moving the context object outside of the component will be a problem.